package.json

```diff
@@ -37,9 +37,11 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "phantomjs-prebuilt": "^2.1.7",
-    "react": "^15.0.0",
```

Comment:
`peerDependencies` aren't automatically installed in npm v3+, so this probably needs to stay in `devDependencies` as well so they don't need to be manually installed during development